src/media/js/views/homepage.js

```diff
@@ -6,6 +6,8 @@ define('views/homepage',
     var gettext = l10n.gettext;
     var appsModel = models('apps');
 
+    var $appContextmenu = document.getElementById('contextmenu').children[0];
```

Comment:
This is actually from another bug's commit, but I fixed the naming anyway :)

And I think the race condition will not occur in this case since the contextmenu element is already inside index.html.